commons/src/main/java/com/powsybl/commons/xml/XmlUtil.java

```diff
@@ -173,4 +179,29 @@ public static float readOptionalFloatAttribute(XMLStreamReader reader, String at
         String attributeValue = reader.getAttributeValue(null, attributeName);
         return attributeValue != null ? Float.valueOf(attributeValue) : defaultValue;
     }
+
+    public static XMLStreamWriter writeStartAttributes(OutputStream os, String prefix, String namespaceUri, String rootName, String indentString, boolean indent) throws XMLStreamException {
+        XMLStreamWriter writer;
+        writer = createXmlStreamWriter(indentString, indent, os);
+        writer.writeStartDocument(StandardCharsets.UTF_8.toString(), "1.0");
+        writer.setPrefix(prefix, namespaceUri);
+        writer.writeStartElement(namespaceUri, rootName);
+        writer.writeNamespace(prefix, namespaceUri);
+        return writer;
+    }
+
+    private static XMLStreamWriter createXmlStreamWriter(String indentString, boolean indent, OutputStream os) throws XMLStreamException {
+        XMLStreamWriter writer = XML_OUTPUT_FACTORY_SUPPLIER.get().createXMLStreamWriter(os, StandardCharsets.UTF_8.toString());
+        if (indent) {
+            IndentingXMLStreamWriter indentingWriter = new IndentingXMLStreamWriter(writer);
+            indentingWriter.setIndent(indentString);
+            writer = indentingWriter;
+        }
+        return writer;
+    }
+
+    public static void writeEndElement(XMLStreamWriter writer) throws XMLStreamException {
+        writer.writeEndElement();
+        writer.writeEndDocument();
+    }
```

Comment:
Same question? The name of this method is not OK. As a developer, I don't know it will end my XML document.